go-controller/pkg/util/ovs.go

```diff
@@ -430,13 +430,9 @@ func runOVNretry(cmdPath string, envVars []string, args ...string) (*bytes.Buffe
 	}
 }
 
-var NbctlCounter uint64
```

Comment:
you added this in the first commit, but removing it in teh 2nd, just remove it all together from the commits.